### PR TITLE
Timetable UI feature

### PR DESCRIPTION
**Pull request by alexiscatnip**
**WIP, but Pullable :)**

Added functionality:
- Replaced the BrowserPanel class with TimeTablePanel class.
    - It does nothing now (cant display anything else). Only look nice...
    - It now looks like an actual timetable instead of placeholder text. This class will eventually contain all the other UI classes: TimeTablePanelMainGrid, TimeTablePanelTimeSlots, etc. You can see the morphing direction inside TimeTablePanel for a full understanding.

Removed functionality: 
- Disabled (commented-out) some tests that reference the 'BrowserPanel'.
   - have to create tests for TimeTablePanel instead.

Todo list for this feature (can display timetable):
- Timetable operations:
    - add, edit, delete timeslots
    - show user's timetable
    - show deconflicted-timetable
    - Scale timetable based on user-selected start & end-times.
    - The above implemented with the help of a helper class Inside TimeTablePanel. EG: TimeTablePanelLogic, to manage UI logic, such as populating the timeslots and scaling the timetable.
- tests for TimeTablePanel.

### CODE REVIEW

File: src/main/java/seedu/address/model/person/Person.java [hunk 1]
```diff
@@ -120,4 +120,7 @@ public String toString() {
         return builder.toString();
     }
 
+    public TimeTable getTimeTable() {
+        return  weekly;
```
Comment (nianfei97) [inline src/main/java/seedu/address/model/person/Person.java]: Double space here will probably trigger stylecheck. Also "weekly" as a timetable name doesn't seem to be the best variable name

File: src/test/java/guitests/guihandles/MainWindowHandle.java [hunk 2]
```diff
@@ -12,7 +12,7 @@
     private final CommandBoxHandle commandBox;
     private final StatusBarFooterHandle statusBarFooter;
     private final MainMenuHandle mainMenu;
-    private final BrowserPanelHandle browserPanel;
+    private final TimetablePanelHandle browserPanel;
```
Comment (nianfei97) [inline src/test/java/guitests/guihandles/MainWindowHandle.java]: timeTablePanel?

File: src/test/java/guitests/guihandles/MainWindowHandle.java [hunk 3]
```diff
@@ -45,7 +45,7 @@ public MainMenuHandle getMainMenu() {
         return mainMenu;
     }
 
-    public BrowserPanelHandle getBrowserPanel() {
+    public TimetablePanelHandle getBrowserPanel() {
```
Comment (nianfei97) [inline src/test/java/guitests/guihandles/MainWindowHandle.java]: getTimetablePanel()?

File: src/test/java/guitests/guihandles/MainWindowHandle.java (hunk 2 above)
Comment (alexiscatnip) [reply src/test/java/guitests/guihandles/MainWindowHandle.java]: nice catch

**Pull request merged**